### PR TITLE
NETOBSERV-915 add conversation direction

### PR DESCRIPTION
**Pull request by jpinsonneau**
This PR adds direction fields to conversation tracking

Also, it adds JSON fields sorting to be able to easy compare 2 items

Related PRs:
- https://github.com/netobserv/network-observability-operator/pull/313
- https://github.com/netobserv/flowlogs-pipeline/pull/430

### CODE REVIEW

File: web/src/components/netflow-record/record-panel.tsx [hunk 1]
```diff
@@ -225,6 +225,12 @@ export const RecordPanel: React.FC<RecordDrawerProps> = ({
     }
   }, [record.labels._RecordType, t]);
 
+  const getSortedJSON = React.useCallback(() => {
+    const allKeys = new Set<string>();
+    JSON.stringify(record, (key, value) => (allKeys.add(key), value));
```
Comment (jotak) [inline web/src/components/netflow-record/record-panel.tsx:230]: This `stringify` call seems useless? (return value is ignored)
Could it be simplified with just: 
```
return JSON.stringify(record, Object.keys(record).sort(), 2); 
```
?

Comment (jpinsonneau) [reply web/src/components/netflow-record/record-panel.tsx:230]: that's a trick to list all keys of `record`
https://stackoverflow.com/a/53593328

else you would need to implement an [iterator to get subkeys](https://stackoverflow.com/a/16168003) (record.labels & record.fields)

Comment (jotak) [reply web/src/components/netflow-record/record-panel.tsx:230]: This then: ?

```
  const flat = { ...record.fields, ...record.labels };
  return JSON.stringify(flat, Object.keys(flat).sort(), 2);
```

(sorry but the double-stringify looks really hacky :-) )

Comment (jpinsonneau) [reply web/src/components/netflow-record/record-panel.tsx]: done

**Comment by jpinsonneau** [comment]
Addressed feedback on json sort + include `_RecordType` as secondary criteria to sort start / end time columns to ensure start < tick < end order when logs have exact same time:

image

**Review by jotak — APPROVED** [review]

**Pull request merged**